src/Commands/stubs/Auth/ForgotPasswordController.stub

```diff
@@ -3,34 +3,34 @@
 namespace {{namespace}};
 
 use App\Mail\Restify\Auth\ForgotPasswordMail;
+use App\Models\User;
 use Illuminate\Http\Request;
 use Illuminate\Routing\Controller;
 use Illuminate\Support\Facades\Mail;
 use Illuminate\Support\Facades\Password;
-use App\Models\User;
 
 class ForgotPasswordController extends Controller
 {
     public function __invoke(Request $request)
     {
         $request->validate([
             'email' => ['required', 'email'],
+            'url' => ['sometimes', 'string'],
         ]);
 
         /** * @var User $user */
-
         $user = User::query()->where($request->only('email'))->firstOrFail();
 
         $token = Password::createToken($user);
 
-        $url = str_replace(
+        $passwordResetUrl = str_replace(
```

Comment:
I think this can remain as `$url` ... you are already in a Password Reset context